src/Api/Model/Languageforge/Lexicon/Command/LexUploadCommands.php

```diff
@@ -279,20 +279,37 @@ public static function deleteMediaFile($projectId, $mediaType, $fileName) {
                 $data->errorMessage = $errorMsg;
                 return $response;
         }
-        $filePath = $folderPath . '/' . $fileName;
+
+        //Path to the specific file the entry points to
+        $filePath = "$folderPath/$fileName";
+        //Put any other stored versions of the file (e.g. the same file saved in other formats) in an array
+        $fileNameWithoutExt = preg_replace('/\\.[^.\\s]{3,4}$/', '', $fileName);
+        $versionsOfTheSameFile = glob("$folderPath/$fileNameWithoutExt.*");
+
         if (file_exists($filePath) and ! is_dir($filePath)) {
+
+            //Delete the file the entry points to and create the server response
             if (unlink($filePath)) {
                 $data = new MediaResult();
                 $data->path = $folderPath;
-                $data->fileName = $fileName;
+                $data->fileName = $filePath;
```

Comment:
I am trying to understand why this is changed to $filePath, but I don't see it.  Can you clarify?

Reply:
That was my mistake. The latest commit restores the original use of $fileName.